silverstripe/framework/CVE-2019-19325.yaml

```diff
@@ -0,0 +1,11 @@
+title:     'CVE-2019-19325: XSS through non-scalar FormField attributes'
+link:      https://www.silverstripe.org/download/security-releases/cve-2019-19325/
+cve:       CVE-2019-19325
+branches:
+    4.0.x:
+        time:     2020-02-16 23:54:40Z
```

Comment:
will the 'Z' at the end of time matter? possible remove it? last PR we did did not have 'Z' on the end of the times and that was merged by friends of php
e.g. https://github.com/FriendsOfPHP/security-advisories/commit/9081ae8f256783ee639b938cf29ef9fa0f928144

Reply:
Lol ... copied it from the Github date attribute. I removed the T between the date and the tiem but miss the Z at the end.

Reply:
Fixed.